daemons/ipa-kdb/ipa_kdb_mspac.c

```diff
@@ -417,11 +419,21 @@ static krb5_error_code ipadb_fill_info3(struct ipadb_context *ipactx,
             if (strcasecmp(objectclasses[c], "ipaNTUserAttrs") == 0) {
                 is_user = true;
             }
+            if (strcasecmp(objectclasses[c], "ipaIDObject") == 0) {
+                is_idobject = true;
+            }
+            if (strcasecmp(objectclasses[c], "ipaUser") == 0) {
+                is_ipauser = true;
+            }
             free(objectclasses[c]);
         }
     }
     free(objectclasses);
 
+    if (is_idobject && is_ipauser) {
```

Comment:
Can you add a block comment with a message similar to commit message so the reason for this is clear?

Reply:
Done.

Reply:
BTW my inspiration for suggesting adding it to the code was the great commit message overall! They all really helped me get a handle on what each patch was doing.